ui/desktop/electron-app/src/spawn-session.js

```diff
@@ -0,0 +1,44 @@
+const process = require('process');
```

Comment:
I am not the best when it comes to English and I do not have a strong opinion, but seeing the code comments I would say `spawned` feels more appropriate?

Reply:
I'm increasingly concerned about the growing number of unorganised source files in electron-app.  Something to think about.

Should we have a proper data model/class for sessions?  Maybe refactor the current sessions code to take an OOP approach.  That said, this feels like it should be combined with the code that actual spawns the sessions.  It is more than just a data structure, since it is empowered to kill sessions.